llvm/lib/Target/AArch64/AArch64MachineFunctionInfo.h

```diff
@@ -35,6 +35,11 @@ struct AArch64FunctionInfo;
 class AArch64Subtarget;
 class MachineInstr;
 
+struct TPIDR2Object {
+  unsigned FrameIndex = 0;
```

Comment:
I've found that other places which initialise a frame index use `std::numeric_limits<int>::max()`, so I think it's safer to use this here as well (e.g. see `SwiftAsyncContextFrameIdx`).

Reply:
It looks safer to do what other code does as well, done.